spring-cloud-contract-verifier/src/main/java/org/springframework/cloud/contract/verifier/messaging/kafka/ContractVerifierKafkaStubMessagesInitializer.java

```diff
@@ -29,27 +29,31 @@
 import org.springframework.kafka.test.EmbeddedKafkaBroker;
 import org.springframework.kafka.test.utils.KafkaTestUtils;
 
-class ContractVerifierKafkaStubMessagesInitializer
-		implements KafkaStubMessagesInitializer {
+class ContractVerifierKafkaStubMessagesInitializer implements KafkaStubMessagesInitializer {
```

Comment:
NOTE: I have the latest Spring Cloud formatter and reformat command did the formatting changes. I am assuming this is ok?